lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -480,6 +480,10 @@ def relationships_for(serializer, requested_associations, include_slice)
       #   }.reject! {|_,v| v.nil? }
       def links_for(serializer)
         serializer._links.each_with_object({}) do |(name, value), hash|
+          if value.is_a?(ActiveModel::Serializer::Link)
```

Comment:
Isn't it always a ActiveModel::Serializer::Link now?

Reply:
You are right. Changed it.